### DISCUSSION TITLE
Pinia 3.0

### DISCUSSION THREAD

**Discussion by VasekProchazka** [vuejs/pinia#2855] [Help and Questions] [answered]
Are there already public information or sources about what version 3.0 should look like and what the difference should be compared to 2.0?

**Comment by Deckluhm** [comment, marked as answer]
> In Jan 2025, I will be releasing pinia 3 for Vue 3. With no other breaking change [...]

https://bsky.app/profile/esm.dev/post/3lbktiqj4iz2k

This is from Pinia creator, and from my understanding the only change will be the Vue 3 requirement (so no new feature or bug fix).

**Reply by Deckluhm** [reply]
If you want more technical detail, you can read this post from him too: https://bsky.app/profile/esm.dev/post/3lcifnulidx2t

Basically removing Vue 2 support means Pinia will have a lighter, easier-to-maintain codebase 🧹 ✨

**Reply by posva** [reply]
Thanks! This and also upgrading vue-devtools-api to the latest version too
